modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -59,8 +61,36 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
 ----
 
 * *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
-* *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
-* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
+** *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
++
+When an {OCP} cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. In the Azure CLI, you can display all resource groups by issuing:
++
+[source,terminal]
+----
+az group list
```

Comment:
Just a few changes to make this command formatting follow our docs guidelines:
```suggestion
When an {OCP} cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. In the Azure CLI, you can display all resource groups by issuing the following command:
+
[source,terminal]
----
$ az group list
```

Reply:
You are right
![image](https://github.com/openshift/openshift-docs/assets/106804821/f5ac5e3f-e9ab-4f45-ab26-fff3991010ad)